sdk/communication/Azure.Communication.Administration/tests/SessionRecords/CommunicationIdentityClientLiveTests/IdentityClientFromTokenWithMultipleScope.json

```diff
@@ -0,0 +1,73 @@
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Authorization": "Sanitized",
+        "traceparent": "00-29f56b48ea76234cb552eba4eba3bdd2-b8cefdc9f6d3b54b-00",
+        "User-Agent": "azsdk-net-Communication.Administration/1.0.0-alpha.20201211.1 (.NET Framework 4.8.4250.0; Microsoft Windows 10.0.19042 )",
+        "x-ms-client-request-id": "b31f264ccb3f2decec3bf9a97f60489c",
+        "x-ms-return-client-request-id": "true"
+      },
+      "RequestBody": null,
+      "StatusCode": 200,
+      "ResponseHeaders": {
+        "api-supported-versions": "2020-01-15-preview3, 2020-07-20-preview1, 2020-07-20-preview2",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Fri, 11 Dec 2020 18:28:35 GMT",
+        "MS-CV": "cxlwOH8Pkk6KApXRGglC3g.0",
+        "Strict-Transport-Security": "max-age=2592000",
+        "Transfer-Encoding": "chunked",
+        "X-Azure-Ref": "0U7rTXwAAAACJS7Y9Hj\u002B5QaU9OeCKwTezRVdSMzBFREdFMDYwNwA5ZmM3YjUxOS1hOGNjLTRmODktOTM1ZS1jOTE0OGFlMDllODE=",
+        "x-ms-client-request-id": "b31f264ccb3f2decec3bf9a97f60489c",
+        "X-Processing-Time": "208ms"
+      },
+      "ResponseBody": {
+        "id": "Sanitized"
+      }
+    },
+    {
+      "RequestUri": "https://sanitized.communication.azure.com/identities/sanitized/token?api-version=2020-07-20-preview2",
+      "RequestMethod": "POST",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Authorization": "Sanitized",
+        "Content-Length": "26",
+        "Content-Type": "application/json",
+        "traceparent": "00-0b7630e26f12c747b46b45de4d3db74e-954989e89e10e046-00",
+        "User-Agent": "azsdk-net-Communication.Administration/1.0.0-alpha.20201211.1 (.NET Framework 4.8.4250.0; Microsoft Windows 10.0.19042 )",
+        "x-ms-client-request-id": "4fc8c763d0f9de4edc8e12f2c4f4dd22",
+        "x-ms-return-client-request-id": "true"
+      },
+      "RequestBody": {
+        "scopes": [
+          "chat",
+          "pstn"
+        ]
+      },
+      "StatusCode": 200,
+      "ResponseHeaders": {
+        "api-supported-versions": "2020-01-15-preview3, 2020-07-20-preview1, 2020-07-20-preview2",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Fri, 11 Dec 2020 18:28:35 GMT",
+        "MS-CV": "z8ONAMo1BU22O/\u002BUm\u002BZs9w.0",
+        "Strict-Transport-Security": "max-age=2592000",
+        "Transfer-Encoding": "chunked",
+        "X-Azure-Ref": "0U7rTXwAAAAA9FAWi5r5BRbUWSerJ\u002B3l5RVdSMzBFREdFMDYwNwA5ZmM3YjUxOS1hOGNjLTRmODktOTM1ZS1jOTE0OGFlMDllODE=",
+        "x-ms-client-request-id": "4fc8c763d0f9de4edc8e12f2c4f4dd22",
+        "X-Processing-Time": "280ms"
+      },
+      "ResponseBody": {
+        "id": "Sanitized",
+        "token": "Sanitized",
+        "expiresOn": "2020-12-12T18:28:35.0277177\u002B00:00"
+      }
+    }
+  ],
+  "Variables": {
+    "COMMUNICATION_ENDPOINT_STRING": "https://test-communication.communication.azure.com/",
```

Comment:
We should sanitize COMMUNICATION_ENDPOINT_STRING

Reply:
Let's add `JsonPathSanitizers.Add("$..COMMUNICATION_ENDPOINT_STRING");` to https://github.com/Azure/azure-sdk-for-net/blob/13aff100da135d11cd593965e3397b899ac5807d/sdk/communication/Azure.Communication.Common/tests/CommunicationRecordedTestSanitizer.cs#L28

Reply:
Added!